src/widgets/span/lv_span.h

```diff
@@ -214,6 +228,38 @@ uint32_t lv_spangroup_get_expand_width(lv_obj_t * obj, uint32_t max_width);
  */
 int32_t lv_spangroup_get_expand_height(lv_obj_t * obj, int32_t width);
 
+/**
+ * Get the span's coords in the spangroup.
+ * @note Before calling this function, please make sure that the layout of span group has been updated.
+ * Like calling lv_obj_update_layout() like function.
+ *     +--------+
+ *     |Heading +--->------------------+
+ *     |  Pos   |   |     Heading      |
```

Comment:
I am sorry about the ambiguity of my earlier comment about indenting 4 spaces would cause Doxygen to preserve it.  There is one more thing you have to do to achieve that:  add a blank line below the paragraph above it.  I did a test with that exact diagram and without the blank line, Doxygen tries to make it part of the paragraph above it and does line wrapping, and tries to interpret the lines as meaning a couple of different things not intended.

![image](https://github.com/user-attachments/assets/c52219a4-4538-404f-b102-61d66327d547)

However, WITH the blank line, it preserves it as intended:

![image](https://github.com/user-attachments/assets/3c8ec2fe-7e03-410e-8b7d-410ad878779d)

Reply:
how to do?

Reply:
@vwheeler63 Can you send how should it look like?

Reply:
> @vwheeler63 Can you send how should it look like?

Certainly!  This blank line prevents Doxygen from trying to interpret the diagram as being part of the paragraph above it.

```
 * Like calling lv_obj_update_layout() like function.
 *
 *     +--------+
 *     |Heading +--->------------------+
 *     |  Pos   |   |     Heading      |
```

Reply:
I wanna know how to build docs locally. Because maybe I need to write many docs for cache and associated objects.